UniversityofHelsinki/ResearchDataServiceChannel
# Wizard fixes

to test:

drush cr

Verify that:
- Breadcrumbs are visible
- Breadcrumbs are clickable
- Breadcrumbs are always shown (with base of 'wizard')
- Terms are displayed in correct order ( check from the backend and add one term to custom location)
- Terms are displayed in correct order after traversing wizardi for a bit.
